### PR TITLE
Update StringExtensions.bas

### PR DESCRIPTION
**Pull request by InExSu**
Brought variable declaration out of loop

### CODE REVIEW

File: src/Modules/StringExtensions.bas [hunk 1]
```diff
@@ -3,8 +3,10 @@ Public Function TrimLeft(text As String, c As String) As String
     Dim textLength As Long
     textLength = Len(text)
     
+    Dim firstCharacter As String
+    
     Do While textLength > 0
-        Dim firstCharacter As String
+
```
Comment (michaelneu) [inline src/Modules/StringExtensions.bas]: this new line looks a bit weird. could you please remove it?

File: src/Modules/StringExtensions.bas [hunk 2]
```diff
@@ -23,8 +25,10 @@ Public Function TrimRight(text As String, c As String) As String
     Dim textLength As Long
     textLength = Len(text)
     
+    Dim lastCharacter As String
+        
     Do While textLength > 0
-        Dim lastCharacter As String
+
```
Comment (michaelneu) [inline src/Modules/StringExtensions.bas]: same here, please remove the additional whitespace

**Comment by michaelneu** [comment]
Thanks! I didn't see your PR until now, sorry.

File: src/Modules/StringExtensions.bas (hunk 1 above)
Comment (InExSu) [reply src/Modules/StringExtensions.bas]: Do you prefer multiple variable declarations inside a loop?

File: src/Modules/StringExtensions.bas (hunk 2 above)
Comment (InExSu) [reply src/Modules/StringExtensions.bas]: Maybe I'm not paying attention... about the additional spaces in what line is this about?

Comment (michaelneu) [reply src/Modules/StringExtensions.bas]: I'm talking about [this](https://github.com/michaelneu/webxcel/blob/c65604349e4281b5fb5cdf28373e0a1b4a07bec5/src/Modules/StringExtensions.bas#L9) and [this](https://github.com/michaelneu/webxcel/blob/c65604349e4281b5fb5cdf28373e0a1b4a07bec5/src/Modules/StringExtensions.bas#L31) newline, which look kinda awkward.

File: src/Modules/StringExtensions.bas (hunk 1 above)
Comment (michaelneu) [reply src/Modules/StringExtensions.bas]: Nope. But removing a blank line doesn't add a new declaration 🙃

**Pull request merged**